src/stories/components/TransparencyCard/TransparencyCard.tsx

```diff
@@ -74,8 +74,10 @@ const FooterWrapper = styled.div<{ isLight: boolean }>(({ isLight }) => ({
 const Row = styled.div<{ hasIcon?: boolean; height?: string }>(({ hasIcon = false }) => ({
   display: 'flex',
   alignItems: hasIcon ? 'flex-start' : 'center',
-
   flex: 1,
+  '& > div > div > div:nth-child(2) > div >div:nth-child(2) > div': {
```

Comment:
please, use another approach to add styles to that `div`, there's too much depth in that selector

Reply:
you are right, the change are done